.travis.yml

```diff
@@ -0,0 +1,42 @@
+language: android
+jdk: oraclejdk8
+cache:
+  directories:
+    - $HOME/.gradle/caches/
+    - $HOME/.gradle/wrapper/
+android:
+  components:
+    # use the latest revision of Android SDK Tools
+    - tools
+    - platform-tools
+
+    # The BuildTools version used by your project
+    - build-tools-23.0.2
+
+    # The SDK version used to compile your project
+    - android-24
+
+    # Additional components
+    - extra-google-google_play_services
+    - extra-google-m2repository
+    - extra-android-m2repository
+
+    # uncomment when we enable espresso tests
+    # - sys-img-armeabi-v7a-android-21
+
+  licenses:
+   - 'android-sdk-license-.+'
+
+# uncomment the following lines to use emulator while runnning expresso tests
+# Emulator Management: Create, Start and Wait
+#before_script:
+#  - echo no | android create avd --force -n test -t android-21 --abi armeabi-v7a
+#  - emulator -avd test -no-skin -no-audio -no-window &
+#  - android-wait-for-emulator
+#  - adb shell input keyevent 82 &
```

Comment:
I am not sure about the context here. Are you suggesting to add server url to `.travis.yml` ?